src/SignalR/perf/benchmarkapps/Crankier/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Crankier
+
+Load testing for ASP.NET Core SignalR
+
+## Commands
+
+### local
```

Comment:
I'd suggest adding a small overview of the `local` command here. Something like "The `local` command launches a set of local worker clients to establish connections to your SignalR server and send messages for the duration of the test."

We should also call out that the client expects the server hub to have a method like this:

```csharp
public void SendPayload(string payload)
{
	// ... your code ...
}
```

Maybe point to [Client.StartTest](https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/perf/benchmarkapps/Crankier/Client.cs#L73) which defines the test behavior to run.

Reply:
Client.StartTest has zero references.  Also I ran the `local` command against my own hub that did not have a `SendPayload` and it worked as expected.  I'll test this further this evening, but I suspect `StartTest` is not actually being called or is being called with `0` message size.

Reply:
Ah, you're probably right. This is a bunch of code for the previous iteration of SignalR that was brought forward and hacked up to work so there are definitely some quirks in here :)

Reply:
I've added the overview, but also confirmed that `SendPayload` is currently not called.